keras_hub/src/models/retinanet/retinanet_object_detector.py

```diff
@@ -18,6 +15,10 @@
     RetinaNetObjectDetectorPreprocessor,
 )
 
+# Check if Keras version is greater than or equal to 2.10.0
+if version.parse(keras.__version__) < version.parse("3.8.0"):
```

Comment:
This is not how we can do this. We need this library to be importable with older versions of Keras. And this file will be imported, and this code block run, on library import!

We need to instead assert the version on usage of symbols we care about.
- Add a function [here](https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/utils/tensor_utils.py#L251) called `assert_bounding_box_support()`.
- Call it here inside __init__, first thing. Also add the same check to anchor generator and box matcher layers.